src/features/MentorPage/MentorChips/ShowMoreChips.tsx

```diff
@@ -0,0 +1,61 @@
+import styled from 'styled-components';
+import ExpandIcon from '../../../static/img/icon-expand.svg';
+
+/**
+ * Show more button style and layout
+ * Should update Skill Chips container when pressed?
+ */
+
+const ShowMoreButtonHandler = () => {
+  const showMoreButton = document.getElementById('skill-chips');
+  if (showMoreButton) {
+    showMoreButton.classList.toggle('show-more');
+  }
+};
+
+const ShowMoreChips = () => {
+  return (
+    <>
+      <MoreContainer>
+        <MoreIcon />
+        <ShowMoreButton onClick={ShowMoreButtonHandler}>
+          Näytä kaikki aiheet
+        </ShowMoreButton>
+      </MoreContainer>
+    </>
+  );
+};
```

Comment:
This is not a react-pattern that I like to see.
It is unclear only by reading this what is going to happen. 
Some css-class changes when we press the button. 

In React instead of flipping css-classnames, it is usually preferable to switch values and show content based on the values. It should be clear by reading the component-file what is happening. This might be too abstracted, dunno. But personally i dont feel this is a good component as it is

Reply:
I can make a code-suggestion later on, maybe tommorrow

Reply:
And also to add: if this is the last thing blocking from merging (like the other issues are fixed and i have not provided an suggestion), then this can be merged. I can also provide a other PR to reactify this small thing. Then you can see the pattern too.